graphql/src/main/java/io/micronaut/configuration/graphql/DefaultGraphQLExecutionInputCustomizer.java

```diff
@@ -35,7 +38,14 @@
      * {@inheritDoc}
      */
     @Override
-    public Publisher<ExecutionInput> customize(ExecutionInput executionInput, HttpRequest httpRequest) {
+    public Publisher<ExecutionInput> customize(ExecutionInput executionInput, HttpRequest httpRequest,
+                                               @Nullable MutableHttpResponse<String> httpResponse) {
+        // we store original request and response, since it might be needed inside data fetchers
+        if (executionInput.getContext() != null && executionInput.getContext() instanceof GraphQLContext) {
```

Comment:
This is not needed. Users should create there own `GraphQLExecutionInputCustomizer` wehere they can choose whether to store the request or response.

Reply:
Ok, I'll remove it.

Reply:
Originally I think maybe many ppl like me don't want to copy-paste own version of `GraphQLExecutionInputCustomizer`, and prefer to have such feature out-of-box. Because in my apps, almost all of the using headers inside GraphQL data fetchers.